indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -416,6 +416,9 @@
   (merge
     {:concept-id (m/stored m/string-field-mapping)
 
+     :native-id m/string-field-mapping
+     :native-id.lowercase m/string-field-mapping
```

Comment:
Thanks Mark. i will wait for @chris-durbin to comment on this because I don't know how doc-field is supposed to be used.